pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1138,6 +1138,8 @@ type BackupSpec struct {
 	UseKMS bool `json:"useKMS,omitempty"`
 	// Specify service account of backup
 	ServiceAccount string `json:"serviceAccount,omitempty"`
+	// CleanData denotes whether to clean backuped data after backup job is deleted
```

Comment:
```suggestion
	// CleanData denotes whether to clean backup data before the object is deleted from the cluster
```

the finalizer implements `pre-delete` hooks.